R/get_feature_wfs.R

```diff
@@ -182,8 +148,50 @@ handle_result_types <- function(result, result_type, property_name, request) {
         paste0(message, "\nThe requested url was: %s"),
         request
       ))
-    } else {
-      stop(sprintf("Exited with HTTP status code %s", result$status_code))
     }
+    stop(sprintf("Exited with HTTP status code %s", result$status_code))
+  }
+
+  if (result_type == "hits") {
+    parsed <- as_list(content(result, "parsed", encoding = "UTF-8"))
+    n_features <- attr(parsed$FeatureCollection, "numberMatched")
+    return(n_features)
+  }
+
+  content <- content(result, encoding = "UTF-8")
+  # Write the content to disk
+  destfile <- paste0(tempfile(), ".gml")
```

Comment:
Maybe set the default of the `destfile` argument. Then you can omit this line.

```
store_as_gml.raw <- function(content, destfile = tempfile(fileext = "gml"), ...) {
```

Reply:
done!

Reply:
Oops, that was not possible b/c destfile object is used further down in the code

Reply:
> Oops, that was not possible b/c destfile object is used further down in the code

You can solve that by having `store_as_gml()` return the `destfile` object.